app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -400,5 +402,11 @@ class SettingsFragment : PreferenceFragmentCompat(), AccountObserver {
             preferenceFirefoxAccountAuthError?.isVisible = false
             accountPreferenceCategory?.isVisible = false
         }
+
+        // if FxA has been turned off on the server
+        if (context?.isInExperiment(asFeatureFxADisabled)!!) {
```

Comment:
@rfk this hides the fxa sign in button

Reply:
So just to confirm this will hide the "sign in" option for new users, but you will still get all the account-management UI if you're already signed in, right?

Reply:
That's the goal yeap